app/build.gradle

```diff
@@ -131,12 +131,12 @@ dependencies {
     kapt "com.google.dagger:hilt-compiler:$hilt_version"
 
     // Tor
-    implementation "org.briarproject:onionwrapper-android:0.0.6"
-    tor 'org.briarproject:tor-android:0.4.7.15'
+    implementation "org.briarproject:onionwrapper-android:0.0.7"
```

Comment:
0.1.0 will be available in a few minutes, shall we jump straight to that? There's some small API breakage for TorManager.

Reply:
Oh sorry, missed your comment above.